Source/Core/Core/Debugger/OSThread.cpp

```diff
@@ -192,9 +192,10 @@ std::string OSThreadView::GetSpecific() const
 
   for (u32 addr : m_thread.specific)
   {
-    if (!PowerPC::HostIsRAMAddress(addr))
+    const auto name = PowerPC::HostGetString(addr);
+    if (!name)
       break;
-    specific += fmt::format("{:08x} \"{}\"\n", addr, PowerPC::HostGetString(addr));
+    specific += fmt::format("{:08x} \"{}\"\n", addr, *name);
```

Comment:
Not related to this PR directly but this function could use a `fmt::memory_buffer` to avoid some of the overhead of string concatenation.